src/main/java/org/folio/rest/impl/PurchaseOrderHelper.java

```diff
@@ -290,14 +290,13 @@ protected Errors getProcessingErrors() {
     return completedFuture(null);
   }
 
-  private CompletableFuture<Boolean> validateVendor(CompositePurchaseOrder compPO) {
+  private CompletableFuture<Boolean> validateVendorsAndAccessProviders(CompositePurchaseOrder compPO) {
     if (compPO.getWorkflowStatus() == WorkflowStatus.OPEN) {
-      VendorHelper vendorHelper = new VendorHelper(lang, httpClient, okapiHeaders, ctx);
+      VendorHelper vendorHelper = new VendorHelper(httpClient, okapiHeaders, ctx, lang);
       return vendorHelper
         .validateVendor(compPO)
-        .thenCombine(vendorHelper.validateAccessProviders(compPO), (vendorErrors, accessProvidersErrors) -> {
-          addProcessingErrors(vendorErrors.getErrors());
-          addProcessingErrors(accessProvidersErrors.getErrors());
+        .thenCombine(vendorHelper.validateAccessProviders(compPO), (rVoid, errors) -> {
```

Comment:
Taking into account that results of the `validateVendor` method are not used here explicitly, no sense to use `thenCombine`...